src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/JdbcTestIT.java

```diff
@@ -118,15 +118,16 @@ public void stringOperatorNameCaseInsensitiveTest() {
 
     @Test
     public void dateFunctionNameCaseInsensitiveTest() {
-        assertEquals(
+        assertTrue(
```

Comment:
why change to assertTrue?

Reply:
Becase response object comparison does not ignore letter case in schema.name. Thus I used assertTrue together with .equalsIgnoreCase.